h/accounts/models.py

```diff
@@ -136,6 +147,8 @@ def _get_password(self):
         return self._password
 
     def _set_password(self, raw_password):
+        if not PASSWORD_MIN_LENGTH <= len(raw_password) <= PASSWORD_MAX_LENGTH:
```

Comment:
Ok. there was a 256-character limit on the _hashed_ password (for some reason), but I guess these hashes don't generally change in length as the password does. I'll just remove this